scripts/integration-test.sh

```diff
@@ -0,0 +1,56 @@
+CURRENT_DIR=$(pwd)
```

Comment:
suggestion to separate shell scripts and go code into separate folders, it's quite difficult to identify what to run and how, maybe add the info regarding scripts from PR description into a small README.md inside scripts folder

Reply:
Applied suggestion and split code in different folders.
Also added a README.md